account_commission/models/account_move.py

```diff
@@ -143,16 +143,23 @@ def _compute_agent_ids(self):
                 [("invoice_line_id", "=", res.id)]
             )
             for line in settlement_lines:
-                line.date = False
-                line.agent_id = False
-                line.settled_amount = 0.00
-                line.currency_id = False
-                line.commission_id = False
+                line.update(
```

Comment:
Since v13, the efficiency of assigning with active record pattern (dot notation) once each time is the same as using `update` for assigning all in one operation, so no need of this change.

Reply:
:thinking: interesting, didn't know that
Reverted

Reply:
Yes, it's due to 2 changes in the ORM layer:

- Computed fields calculation postponed till they are used.
- Deferred writing, done only at the end.